grouper/models/user.py

```diff
@@ -0,0 +1,82 @@
+import logging
+
+from sqlalchemy import Boolean, Column, Integer, String
+from sqlalchemy.ext.hybrid import hybrid_property
+from sqlalchemy.orm import relationship
+
+from grouper.constants import GROUP_ADMIN, MAX_NAME_LENGTH, PERMISSION_ADMIN, USER_ADMIN
+from grouper.models.audit_log import AuditLog
+from grouper.models.base.model_base import Model
+from grouper.models.comment import CommentObjectMixin
+from grouper.models.counter import Counter
+from grouper.models.user_token import UserToken
+from grouper.plugin import get_plugins
```

Comment:
Makes sense. Yeah, I kind of ignore `Counter` through all my refactoring cause it was somewhat harmless. It doesn't really call out to anything. I suspect we'll want to move `CommentObjectMixin` to it's own thing at some point but let's leave it until it becomes a problem. :)